practical1/lib/tab2.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:practical1/models/userdata.dart';
+import 'package:hive/hive.dart';
+import 'package:hive_flutter/hive_flutter.dart';
+
+class Tab2 extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return WatchBoxBuilder(
```

Comment:
`WatchBoxBuilder` is depricated. Learn how reactive listening happens in new suggested way from Hive Package